sdk/python/packages/flet/src/flet/app.py

```diff
@@ -35,75 +35,78 @@ def app(*args, **kwargs):
 
 
 @deprecated("Use run() instead.", version="0.70.0", show_parentheses=True)
-async def app_async(*args, **kwargs):
-    return await run_async(*args, **kwargs)
+def app_async(*args, **kwargs):
+    return run_async(*args, **kwargs)
 
 
 def run(
-    target,
+    main,
+    before_main=None,
     name="",
     host=None,
     port=0,
     view: Optional[AppView] = AppView.FLET_APP,
     assets_dir="assets",
     upload_dir=None,
     web_renderer: WebRenderer = WebRenderer.AUTO,
-    use_color_emoji=False,
     route_url_strategy="path",
     no_cdn=False,
     export_asgi_app=False,
+    target=None,  # for backward compatibility
```

Comment:
whats the plan with `target` ? To be removed in the future? If yes, then we can show a deprecation warning if explicitly set, so users can migrate to `main`.

Reply:
It looks like `ft.app(target=main)` worked because it was the first parameter. It doesn't work now and requires `main=main`. Looks like it's going to be a breaking change as I really want to get rid of `target` coming from threads.

Reply:
Added a note for breaking change.